adminSiteClient/EditorHistoryTab.tsx

```diff
@@ -60,6 +62,25 @@ export class EditorHistoryTab extends React.Component<{ editor: ChartEditor }> {
         grapher.rebuildInputOwidTable()
     }
 
+    @action.bound copyYamlToClipboard() {
+        // Use the Clipboard API to copy the config into the users clipboard
+        const chartConfigObject = {
+            ...this.props.editor.grapher.object,
+        }
+        delete chartConfigObject.id
+        delete chartConfigObject.dimensions
+        delete chartConfigObject.version
+        delete chartConfigObject.isPublished
+        const chartConfigAsYaml = dump(chartConfigObject)
+        navigator.clipboard.writeText(chartConfigAsYaml)
```

Comment:
`navigator.clipboard` is undefined at HTTP site (i.e. our staging servers) and would require some workaround. We probably won't need it there, so it's fine to only make it work for production grapher, up to you.

Here's chatgpt snippet with the workaround in case
```javascript
function copyToClipboard(text) {
  if (navigator.clipboard) {
    navigator.clipboard.writeText(text).then(() => {
      console.log('Text copied to clipboard');
    }).catch(err => {
      console.error('Failed to copy text to clipboard', err);
    });
  } else {
    const textarea = document.createElement('textarea');
    textarea.value = text;
    document.body.appendChild(textarea);
    textarea.focus();
    textarea.select();

    try {
      document.execCommand('copy');
      console.log('Text copied to clipboard');
    } catch (err) {
      console.error('Failed to copy text to clipboard', err);
    }

    document.body.removeChild(textarea);
  }
}

```

Reply:
Nice catch!